IdentityCore/src/cache/metadata/MSIDAccountMetadataCacheAccessor.m

```diff
@@ -130,38 +168,44 @@ - (BOOL)updateSignInStateForHomeAccountId:(NSString *)homeAccountId
         return NO;
     }
     
-    MSIDAccountMetadataCacheItem *accountMetadataItem;
-    MSIDAccountMetadataCacheKey *key = [[MSIDAccountMetadataCacheKey alloc] initWitHomeAccountId:homeAccountId clientId:clientId];
+    NSError *localError;
+    MSIDAccountMetadataCacheItem *cacheItem;
```

Comment:
very nit: this can be removed and combined with line 174

Reply:
yep, fixed :P